code/modules/mob/living/silicon/robot/robot_helpers.dm

```diff
@@ -1,7 +1,7 @@
 /mob/living/silicon/robot/set_intent(var/set_intent)
 	a_intent = set_intent
 	cut_overlay(eye_overlay)
-	if(!stat)
+	if(!stat && !isDrone(src))
```

Comment:
You really, really should never be doing `istype(src, ...)`.

Reply:
Can you give me any idea of what I'd use here please? 👀

Reply:
The reason this is here is because otherwise the Drones are placed onto the 'borg overlay system, which causes the base robot overlay to get placed onto them and then immediately blanked. This is meant to stop all Drones from being in the 'borg overlay system, and only in their own (Where when emagged they get turned red eyes)

Reply:
Basically all I want it to do is exclude drones from this thing.

Reply:
O I fixed it but it's through such a minimal way, I had hoped to port the drones to the new system after thinking about it, but it proved to be too much work rn with all of the PRs I have up, so I'm just going to settle for fixing the bug and maybe return to it in v4 if that's okay?

Re-open (Or make another issue) if you see a problem with this or something else.